chandra_time/Time.py

```diff
@@ -2,9 +2,17 @@
 """
 Convert between various time formats relevant to Chandra.
 
-Chandra.Time provides a simple interface to the C++ time conversion
+Chandra_time provides a simple interface to the C++ time conversion
```

Comment:
Do we care about case?  Maybe this should also be chandra_time but that's a nit.

Reply:
Fixed.